apps/docs/src/pages/brand/storybook.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * This page is only used during local development to redirect Storybook links to the Storybook dev server
+ * On production this directory is redirected to the privately hosted Storybook by primer.style
+ */
+
+import {useEffect} from 'react'
+
+const StorybookRedirect = ({location}) => {
+  useEffect(() => {
+    window.location.replace(`http://localhost:6006${location.search}`)
```

Comment:
Hardcoded port because it's hardcoded in `package.json` and in a bunch of test files